src/extension/background-script/router.ts

```diff
@@ -32,7 +32,7 @@ const routes = {
   signMessage: ln.signMessage,
   makeInvoice: ln.makeInvoice,
   connectPeer: ln.connectPeer,
-  getPayments: payments.all,
+  getPayments: payments.listByAccount,
```

Comment:
This now does the same as `getPaymentsByAccount`. Should one of these be removed?

Reply:
Exactly, it will be removed.

Reply:
I removed it now.